src/openrct2/platform/platform.h

```diff
@@ -35,6 +35,10 @@
 #define MAX_PATH 260
 #endif
 
+#ifndef MAX_COMMAND_LENGTH
+#define MAX_COMMAND_LENGTH 2 * MAX_PATH
```

Comment:
I'm going to refactor this to reflect your comments because I agree MAX_COMMAND_LENGTH is a bit vague. and we can always change it later.